tools/tertiary-analysis/seurat/seurat_run_pca.xml

```diff
@@ -46,11 +46,27 @@ seurat-run-pca.R
 
     <tests>
         <test>
-            <param name="rds_seurat_file" ftype="rdata" value="scaled_seurat.rds"/>
-            <output name="rds_seurat_file" ftype="rdata" value="pca_seurat.rds" compare="sim_size" delta="10000000"/>
-            <output name="output_embed" value="pca_embeddings.csv" compare="sim_size" delta="10000000" />
-            <output name="output_load" value="pca_loadings.csv" compare="sim_size" delta="10000000" />
-            <output name="output_sdev" value="pca_stdev.txt" compare="sim_size" delta="10000000" />
+            <param name="rds_seurat_file" ftype="rdata" value="E-MTAB-6077-3k_features_90_cells-scaled.rds"/>
+            <output name="rds_seurat_file" ftype="rdata">
+              <assert_contents>
+                <has_size value="5052296" delta="200000"/>
```

Comment:
I noticed that many of the tests have a `has_size` assertion, with different values but all with `delta="200000"`, even when the size value is close to 200000.  What is the advantage of using this over `delta_frac`?

Reply:
I wasn't aware of delta_frac! Sounds useful, will use it next time. I would normally go and change this PR after such a comment, but this is work for which I have run out of funding partly, so I just need to close it quickly.